### ISSUE TITLE
mock for fs and net?

### ISSUE THREAD

**Issue by SupremeTechnopriest** [calvinmetcalf/rollup-plugin-node-builtins#21]
Could we add some mocks for libraries like `fs` and `net`?  Here is how `browserify` does it https://github.com/substack/node-browserify/blob/master/lib/builtins.js#L12

**Comment by SupremeTechnopriest** [comment]
we can use https://www.npmjs.com/package/browserify-fs for fs like you do here for crypto https://github.com/calvinmetcalf/rollup-plugin-node-builtins/blob/master/src/index.js#L35

**Comment by SupremeTechnopriest** [comment]
working on a pr now

**Comment by calvinmetcalf** [comment]
the mocks yes!, browserify fs um lets not do that by default, that's a whole lot of stuff there

**Comment by SupremeTechnopriest** [comment]
yeah i was actually thinking of making process an option as well.  Im having problems with it looking for `global` in the browser.  Checking to see if upgrading the packages fixes the issue first.

**Comment by SupremeTechnopriest** [comment]
@calvinmetcalf https://github.com/calvinmetcalf/rollup-plugin-node-builtins/pull/22 submitted

**Comment by calvinmetcalf** [comment]
> Yeah I was actually thinking of making process an option as well. Im having problems with it looking for global in the browser. Checking to see if upgrading the packages fixes the issue first.

check the readme process needs the globals shim

**Comment by SupremeTechnopriest** [comment]
submitted #23
